Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN echo "*** Installing Compiler Explorer ***" \
         nodejs \
         make \
         git \
+        clang-11 \
+        libstdc++6 \
```

Comment:
we don't need this one either?

Reply:
I can remove it, just FYI I was getting errors when it wasn't included. The container was missing the libraries.

Reply:
I'll add it to the parent image then!